src/components/sbb-pearl-chain-vertical-item/sbb-pearl-chain-vertical-item.scss

```diff
@@ -15,56 +15,56 @@
   --sbb-pearl-chain-vertical-item-border-width-thin: var(--sbb-border-width-1x);
   --sbb-pearl-chain-vertical-item-spacing-small: var(--sbb-spacing-fixed-1x);
   --sbb-pearl-chain-vertical-item-spacing-medium: var(--sbb-spacing-fixed-4x);
-  --sbb-pearl-chain-vertical-item-color: var(--sbb-pearl-chain-dot-color);
-  --sbb-pearl-chain-vertical-item-color-warning: var(--sbb-pearl-chain-dot-color-warning);
-  --sbb-pearl-chain-vertical-item-color-past: var(--sbb-pearl-chain-dot-color-past);
-  --sbb-pearl-chain-vertical-item-color-walk: var(--sbb-pearl-chain-dot-color-walk);
+  --sbb-pearl-chain-vertical-item-color: var(--sbb-pearl-chain-bullet-color);
+  --sbb-pearl-chain-vertical-item-color-incidence: var(--sbb-pearl-chain-bullet-color-incidence);
```

Comment:
According to https://www.trinka.ai/blog/incident-vs-incidence/#:~:text=What%20is%20the%20difference%20between%20incident%20and%20incidence%3F&text=Incident%20means%20an%20event%2C%20an,the%20rate%20of%20an%20occurrence. incidence is the wrong term, should be indicent?

Reply:
But it should be called disruption anyways ;)